app/models/container_image.rb

```diff
@@ -59,5 +62,13 @@ def perform_metadata_scan(ost)
     scan_via_miq_vm(miq_cnt_group, ost)
   end
 
+  def tenant_identity
+    User.super_admin
```

Comment:
@moolitayer It yields the super_admin object to the block and then returns that object after the block.  See [Object#tap documentation](http://ruby-doc.org/core-2.3.0/Object.html#method-i-tap).

It is equivalent to 
```ruby
user = User.super_admin
user.current_group = Tenant.root_tenant.default_miq_group
user
```